static/css/style.css

```diff
@@ -173,19 +172,19 @@ strong {
 
 /* member */
 
-.sleepy_cat { 
+.sleepy_cat {
 	font-size: 12px;
```

Comment:
Wouldn't it make more sense of defining something like `.member` to add to the all the tags in HTML and then define the `font-size: 12px;` to that?

This is a school case of spaghetti code in CSS!